inter-page/marketing-bot/app.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Copyright 2018-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ *
+ * Starter Project for Marketing bot of the Messenger Platform Inter-bot Tutorial
+ *
+ * Use this project as the starting point for following the tutorial.
+ *
+ * TODO: Waiting for link
+ *
+ */
+
+'use strict';
+
+// Imports dependencies and set up http server
+const
+    request = require('request'),
+    express = require('express'),
+    body_parser = require('body-parser'),
+    ACCESS_TOKEN = process.env.ACCESS_TOKEN,
```

Comment:
standardize. Either capitalize your constants or don't

Reply:
This has been the pattern in other examples and posts, but I will standardise for now.